Sources/PT.PM.Matching/PatternRoot.cs

```diff
@@ -149,7 +149,7 @@ private static void TraverseChildren(PatternUst patternUst, Ust ust, MatchContex
                 }
                 else
                 {
-                    var match = new MatchResult(ust.CurrentSourceFile, context.PatternUst, context.Locations);
+                    var match = new MatchResult(ust, context.PatternUst, context.Locations);
```

Comment:
And what if the positions of the text spans do not correspond to the current `ust`, for example, if we have `PatternAny`, where the matched node may be the `root`.

Reply:
How would I know? As you see, it was programmed like that long before my change. :)